libraries/AP_ESC_Telem/AP_ESC_Telem.cpp

```diff
@@ -607,8 +652,30 @@ void AP_ESC_Telem::update()
                     error_rate  : rpmdata.error_rate
                 };
                 AP::logger().WriteBlock(&pkt, sizeof(pkt));
-                _last_telem_log_ms[i] = telemdata.last_update_ms;
-                _last_rpm_log_us[i] = rpmdata.last_update_us;
+
```

Comment:
Since this is specific to APD I think we should consider having a define to compile it out.

Reply:
Compiler directive - AP_EXTENDED_ESC_TELEM_ENABLE is now introduced to handle this.